test/d2l-applied-filters/d2l-applied-filters.html

```diff
@@ -142,65 +142,79 @@
 					options[8].click();
 
 					afterNextRender(options[1], () => {
-						assert.equal(clearFilters.hidden, false);
+						assert.equal(filterList.hasAttribute('show-clear-list'), true);
 						done();
 					});
 				});
 				test('clear filters button is hidden when 0 filters are applied', function(done) {
-					const clearFilters = appliedFilters.shadowRoot.querySelector('#d2l-clear-filters-button');
+					const filterList = appliedFilters.shadowRoot.querySelector('d2l-labs-multi-select-list');
 					const options = dropdown.querySelectorAll('d2l-filter-dropdown-option');
 
 					options[0].deselect();
 					options[3].deselect();
 					options[6].deselect();
 
 					afterNextRender(options[0], () => {
-						assert.equal(clearFilters.hidden, true);
+						assert.equal(filterList.hasAttribute('show-clear-list'), false);
 						done();
 					});
 				});
 				test('clear filters button is hidden when 1 filters is applied', function(done) {
-					const clearFilters = appliedFilters.shadowRoot.querySelector('#d2l-clear-filters-button');
+					const filterList = appliedFilters.shadowRoot.querySelector('d2l-labs-multi-select-list');
 					const options = dropdown.querySelectorAll('d2l-filter-dropdown-option');
 
 					options[0].deselect();
 					options[3].deselect();
 
 					afterNextRender(options[0], () => {
-						assert.equal(clearFilters.hidden, true);
+						assert.equal(filterList.hasAttribute('show-clear-list'), false);
 						done();
 					});
 				});
 				test('selecting a 4th filter & clicking clear filters fires a d2l-filter-dropdown-cleared event', function(done) {
-					const clearFilters = appliedFilters.shadowRoot.querySelector('#d2l-clear-filters-button');
+					const filterList = appliedFilters.shadowRoot.querySelector('d2l-labs-multi-select-list');
 					const options = dropdown.querySelectorAll('d2l-filter-dropdown-option');
 
 					const handleClearEvent = () => {
 						document.removeEventListener('d2l-filter-dropdown-cleared', handleClearEvent);
+						assert.equal('true', 'true');
 						done();
 					};
 
 					document.addEventListener('d2l-filter-dropdown-cleared', handleClearEvent);
 
 					options[1].click();
 
-					afterNextRender(clearFilters, () => {
+					afterNextRender(filterList, () => {
+						let clearFilters = undefined;
+						const buttons = filterList.shadowRoot.querySelectorAll('d2l-button-subtle');
+						buttons.forEach(a => {
+							if (a.getAttribute('text') === 'Clear filters' && a.getAttribute('class') !== 'hide') {
+								clearFilters = a;
+							}
+						});
 						clearFilters.click();
 					});
 				});
-				test('selecting a 4th filter & clicking clear filters removes filters & hides clear filters button', function(done) {
```

Comment:
Due to a bug in the demo page, the items are still selected even though they've been cleared and the button sticks around invisible, so we don't have any easy way to detect if you can see it or not.